### PR TITLE
Update with 2016 Information

### PR DESCRIPTION
**Pull request by csexton**
A few things that were updated:
- Slack
- Dates
- Location
- Updated details on stay
- Links
  - Suggest a project
  - Scholarship application
- Bulleted lists

image

/cc: @kalimar @seanmarcia @teresafinn

### CODE REVIEW

File: coc.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Be careful in the words that you choose. We are a community of professionals, an
 - Unwelcome sexual attention.
 - Advocating for, or encouraging, any of the above behavior.
 - Repeated harassment of others. In general, if someone asks you to stop, then stop.
-- If you believe someone is violating the code of conduct, we ask that you report it by emailing [Jam Black](mailto:jam@rubyforgood.com) or [Zander Hill](mailto:zander@rubyforgood.com). Any email received will be kept confidential.
+- If you believe someone is violating the code of conduct, we ask that you report it by emailing [Teresa Finn](mailto:teresa@rubyforgood.com) or [Kalimar Maia](mailto:kalimar@rubyforgood.com). Any email received will be kept confidential.
```
Comment (seanmarcia) [inline coc.md]: I just realized I used rubyforgood.com in the email I sent out. I'll make the switch to rubyforgood.org in subsequent ones. We should change the emails to align with the new url too :)

Comment (csexton) [inline coc.md]: Yep, agreed. I will update.

**Comment by seanmarcia** [comment]
:+1:

**Comment by csexton** [comment]
When do we want to make this public? I am happy to hold off if we want to get more details finalized.

**Pull request merged**